### PR TITLE
Update and rename config.js.example to config.js

### PR DESCRIPTION
**Pull request by kimymt**
Added below:
- whitelist of user
- relays info

### CODE REVIEW

**Comment by Yonle** [comment]
Well, Let me help you out

File: config.js [hunk 1]
```diff
@@ -96,6 +96,7 @@ module.exports = {
   // NOTE: - Require NIP-42 compatible nostr client.
   //         You may use <allowed_event_authors> instead if your client does not support NIP-42.
   authorized_keys: [
+    // "npub1m78s5eqv8l7snc5nnxdvlgue6pt5epgplndtem99quhwyptas7jss2qx53",
```
Comment (Yonle) [inline config.js:99]: Remove `// ` at the beginning

File: config.js [hunk 2]
```diff
@@ -143,7 +144,7 @@ module.exports = {
   // Only for when nostr client requesting server information.
   server_meta: {
     "contact": "unset",
-    "pubkey": "0000000000000000000000000000000000000000000000000000000000000000",
+    "pubkey": "npub1m78s5eqv8l7snc5nnxdvlgue6pt5epgplndtem99quhwyptas7jss2qx53",
```
Comment (Yonle) [inline config.js:147]: Convert the npub into hex string.
You could do that by running the following:
```
bostr hexconverter npub1.....
```

Or:
```
node hexconverter.js npub1.....
```

**Comment by kimymt** [comment]
Thank you so much. I accidentally sent a pull request from my own fork.

**Comment by Yonle** [comment]
> Thank you so much. I accidentally sent a pull request from my own fork.
> 

As long there's no private key being put on, there's nothing really need to worry about.

**Comment by Yonle** [comment]
+ you do not need to fork if you are still using the original code.

**Pull request closed without merging**